app/styles/content.styl

```diff
@@ -0,0 +1,88 @@
+div[role='content']
```

Comment:
`role` should be `contentinfo`, not just `content`: https://www.w3.org/TR/wai-aria/roles#contentinfo